### DISCUSSION TITLE
CompositePanel.Item：JSX元素类型没有任何构造或调用签名

### DISCUSSION THREAD

**Discussion by alphabetabc** [alibaba/designable#243] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/29876424/156724568-a60d519c-12ed-4816-91f0-ba1549fb7753.png)
复现如上图
请大佬帮忙看下应该怎么处理

**Comment by alphabetabc** [comment, marked as answer]
已解决

**Reply by forestMrchen** [reply]
求如何解决的

**Reply by NateYip** [reply]
same question
